ApplicationPattern.yaml

```diff
@@ -5072,6 +5051,65 @@ paths:
+                                    uuid:
+                                      type: string
+                                    profile-name:
+                                      type: string
+                                      enum:
+                                        - 'action-profile-1-0:PROFILE_NAME_TYPE_ACTION_PROFILE'
+                                        - 'file-profile-1-0:PROFILE_NAME_TYPE_FILE_PROFILE'
+                                        - 'integer-profile-1-0:PROFILE_NAME_TYPE_INTEGER_PROFILE'
+                                        - 'response-profile-1-0:PROFILE_NAME_TYPE_GENERIC_RESPONSE_PROFILE'
+                                        - 'string-profile-1-0:PROFILE_NAME_TYPE_STRING_PROFILE'
+                                    response-profile-1-0:response-profile-pac:
+                                      type: object
+                                      required:
+                                        - response-profile-capability
+                                        - response-profile-configuration
+                                      properties:
+                                        response-profile-capability:
+                                          type: object
+                                          required:
+                                            - operation-name
+                                            - field-name
+                                            - datatype
+                                          properties:
+                                            operation-name:
+                                              type: string
+                                            field-name:
+                                              type: object
+                                              minProperties: 1
+                                              maxProperties: 1
+                                              properties:
+                                                concrete-field-name:
+                                                  type: string
+                                                field-name-reference:
+                                                  type: string
+                                            description:
+                                              type: string
+                                            datatype:
+                                              type: string
+                                        response-profile-configuration:
+                                          type: object
+                                          required:
+                                            - value
+                                          properties:
+                                            value:
+                                              type: object
+                                              minProperties: 1
+                                              maxProperties: 1
```

Comment:
Similar to above comment, adding "additionalProperties: false" restriction would help in this situation to restrict entry of incorrect property in request-body